### PR TITLE
Reset Password

### PR DESCRIPTION
**Pull request by daniadore**
Ready

### CODE REVIEW

File: app/helpers/password_resets_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module PasswordResetsHelper
```
Comment (huongnt-2545) [inline app/helpers/password_resets_helper.rb:1]: delete this file if it's not useful

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 <div class="row">
   <div class="col-md-6 col-md-offset-3">
     <%= form_for :session, url: login_path do |f| %>
+
       <%= f.label :email %>
       <%= f.email_field :email, class: "form-control" %>
+
       <%= f.label :password %>
       <%= f.password_field :password, class: "form-control" %>
+       <%= link_to t(:forgot_password), new_password_reset_path %>
```
Comment (huongnt-2545) [inline app/views/sessions/new.html.erb:12]: wrong indent

File: config/webpack/environment.js [hunk 3]
```diff
@@ -1,10 +1,3 @@
 const { environment } = require('@rails/webpacker')
-const webpack = require('webpack')
```
Comment (huongnt-2545) [inline config/webpack/environment.js:2]: why do you remove the code at here?

**Comment by daniadore** [comment]
Ready

File: config/webpack/environment.js [hunk 4]
```diff
@@ -1,10 +1,3 @@
 const { environment } = require('@rails/webpacker')
 const webpack = require('webpack')
-environment.plugins.prepend('Provide',
```
Comment (huongnt-2545) [inline config/webpack/environment.js:3]: don't remove, keep the code from line 4 -> 8

**Comment by daniadore** [comment]
Ready

File: config/webpack/environment.js [hunk 5]
```diff
@@ -1,10 +1,9 @@
 const { environment } = require('@rails/webpacker')
 const webpack = require('webpack')
 environment.plugins.prepend('Provide',
-  new webpack.ProvidePlugin({
-    $: 'jquery/src/jquery',
-    jQuery: 'jquery/src/jquery'
-  })
+new webpack.ProvidePlugin({
+        $: 'jquery/src/jquery',
```
Comment (huongnt-2545) [inline config/webpack/environment.js:5]: wrong indent

**Review by huongnt-2545 — APPROVED** [review]
merged
Some wrong indents, you can fix in the next pull request

**Pull request merged**